plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotProxyGrpcRequestBuilder.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pinot.query;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.pinot.common.proto.Server;
+import org.apache.pinot.common.utils.grpc.GrpcRequestBuilder;
+import org.apache.pinot.spi.utils.CommonConstants;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class PinotProxyGrpcRequestBuilder
+        extends GrpcRequestBuilder
```

Comment:
What's the purpose of extending from GrpcRequestBuilder? There's not an "is a" relationship between the two classes, and this class seems to be overriding every method from the parent that's used.

In fact, the only method that's used from the parent is a call to `setBrokerRequest()`, but the `build()` method does not use that value at all.

Reply:
Good point, Pinot 0.10+ has deprecated the BrokerRequest, so we actually don't have to keep it.
And the PinotProxyGrpcRequestBuilder doesn't need to extend GrpcRequestBuilder.